validate.go

```diff
@@ -268,6 +268,10 @@ func (ctx *ValidationContext) validateSignature(el *etree.Element, sig *types.Si
 	if !bytes.Equal(digest, decodedDigestValue) {
 		return nil, errors.New("Signature could not be verified")
 	}
+	
+	if sig.SignatureValue = nil {
```

Comment:
I think you meant `==`.

Reply:
I noticed that yesterday and force pushed but somehow it's still got 1 = 👀

Reply:
This time the force push worked. 🤷